dist/tools/suit/suit-manifest-generator/suit_tool/compile.py

```diff
@@ -373,7 +373,6 @@ def compile_manifest(options, m):
                 'command-arg' : None
             }))
         else:
-            te = []
```

Comment:
Can you remove this? This is external code.

Reply:
My editor is reporting that the variable is not used (which is true) that's why I removed it.

Anyway, this change will be useless after we merge #15095.

Reply:
Ok, no harm with it I guess.

Reply:
You can run the test for `examples/suit_update` to be sure.